MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -76,6 +76,17 @@ public abstract class BaseStreamingService : BaseApiService
         protected IAuthorizationContext AuthorizationContext { get; private set; }
 
         protected EncodingHelper EncodingHelper { get; set; }
+        protected static NumberFormatInfo _ffmpegTimeDeltaFormat { get; private set; }
+
+        static BaseStreamingService()
+        {
+            _ffmpegTimeDeltaFormat = new NumberFormatInfo()
+            {
+                NumberDecimalDigits = 3,
```

Comment:
Why only three?

Reply:
Because that's what it was originally formatted to in the original code. This can change.